src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -282,9 +282,15 @@ private void addErrorMessage(final XContentBuilder builder, final String message
     }
 
     public static enum ErrorType {
-        NONE("ok"), INVALID_CONFIGURATION("Invalid configuration"), INVALID_PASSWORD("Invalid password"), WRONG_DATATYPE("Wrong datatype"),
-        BODY_NOT_PARSEABLE("Could not parse content of request."), PAYLOAD_NOT_ALLOWED("Request body not allowed for this action."),
-        PAYLOAD_MANDATORY("Request body required for this action."), SECURITY_NOT_INITIALIZED("Security index not initialized");
+        NONE("ok"),
+        INVALID_CONFIGURATION("Invalid configuration"),
+        INVALID_PASSWORD("Invalid password"),
+        WRONG_DATATYPE("Wrong datatype"),
+        BODY_NOT_PARSEABLE("Could not parse content of request."),
+        PAYLOAD_NOT_ALLOWED("Request body not allowed for this action."),
+        PAYLOAD_MANDATORY("Request body required for this action."),
+        SECURITY_NOT_INITIALIZED("Security index not initialized"),
+        INVALID_SALT("Salt must be atleast 16 chars long. If greater, only the first 16 bytes will used for salting.");
```

Comment:
typo: "at least". Will it be better to use "Salt must be 16 bytes long". Note that 16 chars != 16 bytes.

Reply:
We are using UTF-8 encoding for converting to bytes right so each char should be 1 byte ? 
Also this is something that will be returned back to user so thought that user will understand "chars" usage better than "bytes"

Reply:
Only 127 chars can be converted to a single byte using UTF-8. I guess it is not valid to assume that only ASCII will be used for the salt.